scripting/get5.sp

```diff
@@ -1150,9 +1151,6 @@ public Action Event_MatchOver(Event event, const char[] name, bool dontBroadcast
     // trigger it either.
     Stats_ResetGrenadeContainers();
 
-    // Write backup before series score increments
-    WriteBackup();
```

Comment:
I took this out. Per https://github.com/splewis/get5/issues/822 I don't understand why we do it.

Reply:
I don't recall any reason for this; I'm okay with removing it. (Worst case, we discover a bug and can actually document it, but I trust your testing, so I don't expect any issue.)